docs/nodes/validate/add-a-validator.md

```diff
@@ -74,156 +75,332 @@ Once your validator’s start time has passed, you will see the rewards it may e
+
+```js
+const ip: string = "localhost";
+const port: number = 9650;
+const protocol: string = "http";
+const networkID: number = 5;
+```
+
+If your node uses an [elastic IP](../../nodes/build/setting-up-an-avalanche-node-with-amazon-web-services-aws.md#assign-an-elastic-ip), you will need to change the `ip` variable in your network setting.
+
+```js
+const ip: string = "<YOUR-NODES-IP-ADDRESS-HERE>";
+const port: number = 9650;
+const protocol: string = "http";
+const networkID: number = 5;
+```
+
+
+3. Run the example script!
 
 ```sh
-curl -X POST --data '{
-    "jsonrpc": "2.0",
-    "method": "info.getNodeID",
-    "params":{},
-    "id": 1
-}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/info
+avalanchejs $ ts-node examples/info/getNodeID.ts
 ```
 
 The response has your node’s ID:
 
-```json
-{
-  "jsonrpc": "2.0",
-  "result": {
-    "nodeID": "NodeID-LMUue2dBBRWdDbPL4Yx47Ps31noeewJji"
-  },
-  "id": 1
-}
+```sh
+avalanchejs $ NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg
 ```
 
+
 `startTime` and `endTime`
 
 When one issues a transaction to join the Primary Network they specify the time they will enter (start validating) and leave (stop validating.) The minimum duration that one can validate the Primary Network is 2 weeks, and the maximum duration is one year. One can re-enter the Primary Network after leaving, it’s just that the maximum _continuous_ duration is one year. `startTime` and `endTime` are the Unix times when your validator will start and stop validating the Primary Network, respectively. `startTime` must be in the future relative to the time the transaction is issued.
 
-`stakeAmount`
+`delegationFeeRate`
+
+Avalanche allows for delegation of stake. This parameter is the percent fee this validator charges when others delegate stake to them. For example, if `delegationFeeRate` is `10` and someone delegates to this validator, then when the delegation period is over, 10% of the reward goes to the validator and the rest goes to the delegator.
+
+#### Stake Amount
+
+In order to validate the FUJI Network, one must stake AVAX.
+
+The sample code assigns the variable, `stakeAmount` to be the minimum amount of AVAX that a node must stake to become a validator(1 AVAX).
+```ts
+const main = async (): Promise<any> => {
+  const stakeAmount: any = await pchain.getMinStake()
+  ...
+}
+```
+
+To stake a different amount, please replace the code above with a stake amount of 1 AVAX or greater. Below sets a new value of 10,000 AVAX (`10000000000000` GWEI). Value is set in GWEI format where `1,000,000,000` GWEI = 1 AVAX. 
+
+Please follow the Example below for steps to change the stake amount:
+
+Locate this line in the file
+
+```js
+  const unsignedTx: UnsignedTx = await pchain.buildAddValidatorTx(
+    ...
+    stakeAmount.minValidatorStake,
+    ...
+  )
+```
+
+and change the default value to a new `BN` value, for example:
+
+```js
+  const unsignedTx: UnsignedTx = await pchain.buildAddValidatorTx(
+    ...
+    new BN(1e12),
```

Comment:
This is a convoluted way to say that they just need to put a new value for the stake amount as `BN(1e12)`